pkg/cloudprovider/vsphere/util.go

```diff
@@ -22,13 +22,14 @@ import (
 	"strings"
 
 	"github.com/pkg/errors"
+	"k8s.io/cloud-provider-vsphere/pkg/common/provider"
 	klog "k8s.io/klog/v2"
 )
 
 const (
 	// ProviderPrefix is the Kubernetes cloud provider prefix for this
 	// cloud provider.
-	ProviderPrefix = "vsphere://"
```

Comment:
Does this need to change? seems like would always be `vsphere://` anyways.

Reply:
True